02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,72 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
-</body>
+  <head>
+    <title>My Bio</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+   <section>
+    <h1>
+      Travis Schlicke
+    </h1>
+    <p>
+      My address goes here
+    </p>
+  </section>
+    <h3>
```

Comment:
Indentation: After closing `section` the next opening tag needs to be on the same level.